navbar.php

```diff
@@ -19,7 +19,7 @@
         <li><a href="./gallery.php">GALLERY</a></li>
         <li><a href="./lastMeet.php">LAST MEET</a></li>
         <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Sponsors <span class="caret"></span></a>
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">SPONSORS <span class="caret"></span></a>
```

Comment:
This has already been fixed by @uagarwal744 in #49 

Please remove this from this particular PR.